design/one-pager-resource-connectivity-mvp.md

```diff
@@ -638,7 +643,324 @@ parameters:
+  route:
+    - cidrBlock: 0.0.0.0/0
+      gateway:
+        referenceType: apiObject
+        value: id.my-gateway.crossplane-system
+--
+apiVersion: network.aws.crossplane.io/v1alpha1
+kind: RouteTableAssociation
+metadata:
+  namespace: crossplane-system
+  name: my-rt-association
+spec:
+  subnetId:
+    referenceType: apiObject
+    value: id.my-subnet-1.crossplane-system
+  routeTableId:
+    referenceType: apiObject
+    value: id.my-rt.crossplane-system
+--
+apiVersion: network.aws.crossplane.io/v1alpha1
+kind: IamRole
+metadata:
+  namespace: crossplane-system
+  name: my-cluster-role
+spec:
+  nameFormat: clusterRole
+  assumeRolePolicy: |
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Principal": {
+            "Service": "eks.amazonaws.com"
+          },
+          "Action": "sts:AssumeRole"
+        }
+      ]
+    }
+--
+apiVersion: network.aws.crossplane.io/v1alpha1
+kind: IamRolePolicyAttachment
+metadata:
+  namespace: crossplane-system
+  name: my-cluster-role-policy-attachment-1
+spec:
+  policy_arn: arn:aws:iam::aws:policy/EKSClusterPolicy
+  role:
+    referenceType: apiObject
+    value: name.my-cluster-role.crossplane-system
+--
+apiVersion: network.aws.crossplane.io/v1alpha1
+kind: IamRolePolicyAttachment
+metadata:
+  namespace: crossplane-system
+  name: my-cluster-role-policy-attachment-2
+spec:
+  policy_arn: arn:aws:iam::aws:policy/AmazonEKSClusterPolicy
+  role:
+    referenceType: apiObject
+    value: name.my-cluster-role.crossplane-system
+
+### RDS connectivity related managed resources
+--
+apiVersion: network.aws.crossplane.io/v1alpha1
+kind: DbSubnetGroup
+metadata:
+  namespace: crossplane-system
+  name: my-db-subnet-group
+spec:
+  nameFormat: subnetGroup
+  subnetIds:
+    - referenceType: apiObject
+      value: id.my-subnet-1.crossplane-system
+--
+apiVersion: network.aws.crossplane.io/v1alpha1
+kind: SecurityGroup
+metadata:
+  namespace: crossplane-system
+  name: my-rds-sg
+spec:
+  nameFormat: rdsSg
+  vpcId:
+    referenceType: apiObject
+    value: id.my-vpc.crossplane-system
+  description: Cluster communication with worker nodes
+  ingress:
+    - fromPort: 3306
+      toPort: 3306
+      protocol: tcp
+      cidrBlocks:
+        - 0.0.0.0/0
+```
+
+Once all these connectivity managed resources are created, the resource classes
+for `EKSCluster` and `RDSInstance` can be configured as following:
+
+```yaml
+---
+apiVersion: database.aws.crossplane.io/v1alpha1
+kind: RDSInstanceClass
+metadata:
+  name: standard-mysql
+  namespace: crossplane-system
+specTemplate:
+  class: db.t2.small
+  masterUsername: masteruser
+  securityGroups:
+    - referenceType: apiObject
+      value: id.my-rds-sg.crossplane-system
+  subnetGroupName:
+    referenceType: apiObject
+    value: name.my-db-subnet-group.crossplane-system
+  size: 20
+  engine: mysql
+provisioner: rdsinstance.database.aws.crossplane.io/v1alpha1
```

Comment:
`provisioner` should be removed here and `providerRef` and `reclaimPolicy` should be indented as part of `specTemplate`

Reply:
Same for `EKSClusterClass` below.